pages/install-device-page.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, Locator, Page } from '@playwright/test';
```

Comment:
we are adding `type` keyword from TypeScript, please check other pages, it is just safety to know you are importing a type that was added in Playwright docs recently.

Reply:
To be honest I don't know what that line does exactly, I mean, I see it imports stuff but I don't realy know what it needs to import and why. just copied from other tests. but each page has its own sauce.

Reply:
I got rid of it, as it works without.